### PR TITLE
add a FAQ about NTS DoT deadlock

### PR DESCRIPTION
**Pull request by czhang03**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -67,3 +67,12 @@ This is an issue with rpm-ostree image-based systems generally, and not specific
 The functionality that provides this, called GHNS, is disabled by default due to the risk posed by the installation of potentially damaging or malicious scripts. This has caused [real damage](https://blog.davidedmundson.co.uk/blog/kde-store-content/). 
 
 If you still want to enable this functionality, run `ujust toggle-ghns`
+
+#### Cannot connect to the internet after boot
```
Comment (RoyalOughtness) [inline FAQ.md]: this should be in the form of a question. also it should be more specific, like "my system time is way off, why won't it auto update?"

File: FAQ.md [hunk 2]
```diff
@@ -67,3 +67,12 @@ This is an issue with rpm-ostree image-based systems generally, and not specific
...
+
+Please check your system time is correct. 
+Sometimes the system time will be set to a incorrect value after the computer was powered off for a long period of time or a CMOS reset.
```
Comment (RoyalOughtness) [inline FAQ.md]: These two lines are unnecessary

File: FAQ.md [hunk 3]
```diff
@@ -67,3 +67,12 @@ This is an issue with rpm-ostree image-based systems generally, and not specific
...
 
 If you still want to enable this functionality, run `ujust toggle-ghns`
+
+#### Cannot connect to the internet after boot
+
+Please check your system time is correct. 
+Sometimes the system time will be set to a incorrect value after the computer was powered off for a long period of time or a CMOS reset. 
+
+Secureblue enforces [secure DNS connection and network time security (NTS)](https://github.com/secureblue/secureblue/blob/live/README.md#what). 
+If your system time is off by a significant amount, the above security features will cause a deadlock during time sync.
```
Comment (RoyalOughtness) [inline FAQ.md]: "If your system time is off by an excessive amount due to rare conditions like a CMOS reset, it must be reset manually and then chrony will work again. This should never be required except under very rare circumstances."

**Review by RoyalOughtness — CHANGES_REQUESTED** [review]

**Comment by czhang03** [comment]
Thanks for the review! I have reworded the FAQ.

File: FAQ.md [hunk 4]
```diff
@@ -67,3 +67,11 @@ This is an issue with rpm-ostree image-based systems generally, and not specific
 The functionality that provides this, called GHNS, is disabled by default due to the risk posed by the installation of potentially damaging or malicious scripts. This has caused [real damage](https://blog.davidedmundson.co.uk/blog/kde-store-content/). 
 
 If you still want to enable this functionality, run `ujust toggle-ghns`
+
+#### Why Won't My Internet Connect After Boot
```
Comment (RoyalOughtness) [inline FAQ.md]: question mark and don't capitalize every word

File: FAQ.md [hunk 5]
```diff
@@ -67,3 +67,11 @@ This is an issue with rpm-ostree image-based systems generally, and not specific
...
+
+If your system time is off by an excessive amount due to rare conditions like a CMOS reset,
```
Comment (RoyalOughtness) [inline FAQ.md]: put this on one line so the page handles the wrapping

**Review by RoyalOughtness — CHANGES_REQUESTED** [review]

**Review by RoyalOughtness — APPROVED** [review]

**Comment by RoyalOughtness** [comment]
@czhang03 FYI, I bypassed signature verification to get this merged. Please setup commit verification for future pull requests

**Pull request merged**